certified-connectors/TxtSync/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2235 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "TxtSync",
+    "description": "TxtSync enables applications to send and receive global SMS while tracking open rates and delivery reports. Easily make SMS part of your organisations worflow processes through TxtSync. \n\nPlease go to https://txtsync.com/microsoft-flow for 3 simple steps on how to get up and running.  Remember to generate your Client Key and Client Secret as these are used for your Username and Password.",
+    "version": "1.1",
+    "contact": {
+      "name": "Warren Blanchard-Doyle",
+      "url": "https://txtsync.com",
+      "email": "support@txtsync.com"
+    }
+  },
+  "x-ms-connector-metadata": [
+    {
+      "propertyName": "Website",
+      "propertyValue": "https://txtsync.com"
+    },
+    {
+      "propertyName": "Privacy policy",
+      "propertyValue": "https://txtsync.com/terms/privacy"
+    },
+    {
+      "propertyName": "Categories",
+      "propertyValue": "Communication;Productivity"
+    }
+  ],
+  "host": "api.txtsync.com",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/sms/send": {
+      "post": {
+        "responses": {
+          "200": {
+            "description": "default",
+            "schema": {
+              "type": "array",
+              "items": {
+                "type": "object",
+                "properties": {
+                  "FromNumber": {
+                    "type": "string",
+                    "description": "The Sender ID or E164 number to send from",
+                    "title": "From Number",
+                    "x-ms-visibility": ""
```

Comment:
@TxtSync Can you please let us know why `x-ms-visibility` is set to empty string? If you do not want it to add, you can remove it.

Reply:
Applies this comment the similar properties in the swagger

Reply:
@TxtSync Looks like it is not fixed, empty values for x-ms-visibility are not allowed. If you have ay specific reason to keep empty strings, please let us know.